packages/webapp/src/views/land/legal-party-list.html

```diff
@@ -0,0 +1,91 @@
+{% extends 'form-layout.html' %}
+
+{% set noOfLegalParites = legalAgreementParties.length %}
+
+{% if noOfLegalParites > 1 %}
+    {% set pageHeading = "You have added " + noOfLegalParites + " legal parties" %}
+{% else %}
+    {% set pageHeading = "You have added " + noOfLegalParites + " legal party" %}
+{% endif %}
+
+{% block formContent %}
+<div class="govuk-body">
+    {% if err %}
+        {% include "errorbanner.html" %}
+    {% endif %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l" tabindex="-1">
+              {{ pageHeading }}
+            </h1>
+
+            {% for org in legalAgreementParties %}
```

Comment:
This creates a new govSummaryList for every legal agreement party, which messes up the formatting. You should have one govSummaryList with one row per legal agreement party. You can achieve this by passing the row array data already processed into the template from the route handler (i.e. put this logic in the route handler not the template).